source/aggregation-pipeline-builder.txt

```diff
@@ -154,18 +150,31 @@ your chosen view mode:
                
                .. figure:: /images/compass/agg-builder-exclude-stage-example.png 
                   :alt: Aggregation Builder exclude stage example
+         
+         .. step:: Add additional pipeline stages.
+            
+            To add an additional pipeline stage after your last aggregation 
+            stage, click :guilabel:`Add Stage`. To add an aggregation stage 
+            before your most recently added stage, click the :guilabel:`+` icon 
+            above the stage card. 
+            
+            Repeat steps 3 and 4 for each additional stage.
 
-         .. step:: Click :guilabel:`Run` at the top right of the pipeline 
-            builder. |compass-short| returns your results in the document view. 
+            .. note:: 
+
+               You can change the order of pipeline stages by dragging 
+               the header of each stage card.
+
+         .. step:: Run the pipeline. 
             
-            .. warning:: 
-               
-               When you click :guilabel:`Run`, the aggregation runs against the 
-               entire collection.
+            Click :guilabel:`Run` at the top right of the pipeline 
```

Comment:
[suggestion] The user is told to click before they know where! Maybe, "In the top right corner of the pipeline builder, click **Run**." 

Also, consider a newline between the action and the result.